tests/extra/test_job_management.py

```diff
@@ -174,6 +174,8 @@ def test_basic(self, tmp_path, job_manager, job_manager_root_dir, sleep_mock):
             ("job-2022", "finished", "foo"),
         ]
 
+        assert not pd.read_csv(job_db_path)[["cpu", "memory", "duration", "costs"]].isnull().any().any()
```

Comment:
this looks quite cryptic. Can't you assert with real  values. All values are predictable, right?

Reply:
The values are predictable however also well hidden in the _testing.py file (which isn't in the testing folder but still rests under the src of openeo)https://github.com/Open-EO/openeo-python-client/blob/34d42b8267aab2810784d5d61596389b1d0a1a19/openeo/rest/_testing.py#L34

This is why I chose to simply check that the values are not none.
Would it help to add a comment `# check that the columns "cpu", "memory", "duration" and "costs" of finished jobs are not empty` or do you prefer to check on the actual values and add a comment referencing where they are defined? (last option does hold the danger that when the _testing.py file is moved this comment won't be updated)

Third option of course would be to immediately move the _testing.py file in this PR

Reply:
I'd prefer to check for actual values (especially because the job manager also does some formatting of the usage values, which is not really covered by the tests I think).
The values are indeed somewhat hidden, but you can improve the situation by using values that are more obviously dummy values and easier to find, e.g.:

```
     "cpu": {"unit": "cpu-seconds", "value": 1234.5},
     "duration": {"unit": "seconds", "value": 2345},
     "memory": {"unit": "mb-seconds", "value": 56789},
```